spyder/plugins/pythonpath/utils.py

```diff
@@ -18,12 +18,16 @@
 def check_path(path):
     """Check that `path` is not a [site|dist]-packages folder."""
     if os.name == 'nt':
-        pat = re.compile(r'.*lib/(?:site|dist)-packages.*')
+        pattern = re.compile(r'.*(?:l|L)ib/(?:site|dist)-packages.*')
```

Comment:
Is `?:` necessary? I don't think a non-capturing group is necessary in this context, but it doesn't hurt anything either...